asciidoc/courses/importing-data/modules/2-using-data-importer/lessons/2-c-importing-CSV/lesson.adoc

```diff
@@ -41,7 +41,7 @@ Uncompress this file. You should have these files:
 
 == Step 2: Open the Data Importer
 
-In a Web browser window, open the https://data-importer.graphapp.io/?acceptTerms=true[Neo4j Data Importer^]
+In a Web browser window, open the https://data-importer.neo4j.io/versions/0.7.0/?acceptTerms=true[Neo4j Data Importer^]
```

Comment:
Same here - users will have trouble connecting to local databases on the neo4j.io link - is this OK?

Reply:
For the course, they will be connecting to a sandbox, not local